autogpt_platform/frontend/src/components/admin/marketplace/actions.ts

```diff
@@ -3,9 +3,19 @@
 // import ServerSideMarketplaceAPI from "@/lib/marketplace-api/server-client";
 // import { revalidatePath } from "next/cache";
 // import * as Sentry from "@sentry/nextjs";
-// import { checkAuth, createServerClient } from "@/lib/supabase/server";
 // import { redirect } from "next/navigation";
-// import { createClient } from "@/lib/supabase/client";
+
+// export async function checkAuth() {
```

Comment:
Moved from other file.

Reply:
can we remove all of this entirely, if its all just commented out?

Reply:
I think @Bentlybro  is working on this code right now

Reply:
for now leave them